### ISSUE TITLE
Bug: Redis connection refused error

### ISSUE THREAD

**Issue by yhyjerjerry** [MISP/MISP#9814]
### Actual behavior

In v2.4.192, it showed an connection error with redis while in v2.4.191 it does not have.
What I try to do is to publish an event and it will show the error

### Expected behavior

Show the publish event job queued

### Steps to reproduce

Find an event that has not been published
Click the id of the event
Click Publish Event
Click Yes

### Version

2.4.192

### Operating System

Linux

### Operating System version

Amazon linux 2

### PHP version

N/A

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jerryyucrypto** [comment]
![image (44)](https://github.com/MISP/MISP/assets/88696434/39ec570d-e3ea-499f-a8c8-526ce974611b)
Attached error message

**Comment by adulau** [comment]
So if you have a connection refused to Redis: 

- Could you check if your Redis server is running? if not, you will need to start it
- Could you check the hostname and url of the Redis in your configuration? if it's localhost, review it's accessible in IPv6 or IPv4
- Could you check if you don't have any packet filtering on your localhost interface? and check with `tcpdump` if you see your Redis connection on the TCP/6379.

**Comment by jerryyucrypto** [comment]
@adulau Yes Redis is running. I am hosting it in a cluster in a container and is able to connect to it
